### PR TITLE
feature/mobile-search

### PR DESCRIPTION
**Pull request by haritonasty**
**Summary**
- added ability to search trends on mobile
- added assets and trends tabs to search page on mobile

**WARNING** Pls, merge it after [that](https://github.com/santiment/app/pull/386)

**Screenshots**
image
image

### CODE REVIEW

File: src/pages/SearchMobilePage/SearchMobilePage.js [hunk 1]
```diff
@@ -1,21 +1,108 @@
...
+  const [selectedTab, selectTab] = useState(TABS[0].index)
+  const onSelectTab = selected => selectTab(selected)
+  const assetsRecent = getRecentAssets()
+  const assetsWithInfo = assetsRecent.map(asset => ({
+    ...allProjects.find(({ slug }) => slug === asset)
+  }))
+  const [assets, setAssets] = useState(assetsWithInfo)
+  const [trends, setTrends] = useState(getRecentTrends())
+
+  console.log(trends, assets, selectedTab)
```
Comment (Partysun) [inline src/pages/SearchMobilePage/SearchMobilePage.js]: Clean logs

File: src/pages/SearchMobilePage/SearchMobilePage.js [hunk 2]
```diff
@@ -1,21 +1,108 @@
...
+    content: 'Trends'
+  }
+]
+
+const SearchMobilePage = ({ history }) => {
+  const [selectedTab, selectTab] = useState(TABS[0].index)
+  const onSelectTab = selected => selectTab(selected)
+  const assetsRecent = getRecentAssets()
+  const assetsWithInfo = assetsRecent.map(asset => ({
+    ...allProjects.find(({ slug }) => slug === asset)
```
Comment (Partysun) [inline src/pages/SearchMobilePage/SearchMobilePage.js]: If the user opens the project which not contains in saved json, then we can't show it in recent. It's ok now, but we need to save this as a card

File: src/pages/SearchMobilePage/SearchMobilePage.js [hunk 3]
```diff
@@ -1,21 +1,108 @@
...
+  }))
+  const [assets, setAssets] = useState(assetsWithInfo)
+  const [trends, setTrends] = useState(getRecentTrends())
```
Comment (Partysun) [inline src/pages/SearchMobilePage/SearchMobilePage.js]: Why did not use assetsRecent const, which you define above? @haritonasty

File: src/pages/SearchMobilePage/SearchMobilePage.js [hunk 4]
```diff
@@ -1,21 +1,108 @@
...
+      <Tabs
+        options={TABS}
+        defaultSelectedIndex={selectedTab}
+        onSelect={onSelectTab}
+        className={styles.tabs}
+      />
+      <div className={styles.recentWrapper}>
+        <h3 className={styles.caption}>Recently searched</h3>
+        {selectedTab === TABS[0].index &&
+          assets.map(({ name, ticker, coinmarketcapId }) => (
```
Comment (Partysun) [inline src/pages/SearchMobilePage/SearchMobilePage.js]: Can we use our slug instead cmcID?

File: src/pages/SearchMobilePage/SearchMobilePage.js (hunk 3 above)
Comment (Partysun) [inline src/pages/SearchMobilePage/SearchMobilePage.js]: Do we have a limit of size for recent Trends/assets? @haritonasty

Comment (haritonasty) [reply src/pages/SearchMobilePage/SearchMobilePage.js]: yes, 5

File: src/pages/SearchMobilePage/SearchMobilePage.js (hunk 2 above)
Comment (haritonasty) [reply src/pages/SearchMobilePage/SearchMobilePage.js]: I think, better to remove this logic now and use only slugs in recent block

**Pull request merged**